codenames-domain/src/main/java/com/codenames/services/GameService.java

```diff
@@ -65,13 +64,28 @@ public Room getRoomByRequest(CodeNamesGame codeNamesGame, Request<?> request){
         return codeNamesGame.getGameRoom(playerService.getPlayerRoomID(request));
     }
 
+    public int checkRoomAvailable(CodeNamesGame codeNamesGame, int roomID){
```

Comment:
why does it return -1 on error?

Reply:
Created an overused method, I've already deleted it